ui/admin/app/styles/app.scss

```diff
@@ -598,69 +598,71 @@
   }
 }
 
-.ordered-series-diagram {
-  --group-title-line-height: 2.25rem;
+.ordered-series-diagram-wrapper {
+  min-height: 150px;
 
-  width: 100%;
-  display: flex;
-  margin-bottom: 1rem;
-
-  &-item {
+  .ordered-series-diagram {
+    --group-title-line-height: 2.25rem;
+    
     display: flex;
-    margin-top: var(--group-title-line-height);
 
-    &-content {
+    &-item {
       display: flex;
-      flex-direction: column;
-      align-items: center;
-      text-align: center;
-    }
-
-    &-title {
-      font-weight: var(--token-typography-font-weight-semibold);
-      margin-top: 0.5rem;
-      width: 4rem;
-    }
+      margin-top: var(--group-title-line-height);
 
-    &:last-child {
-      > .ordered-series-diagram-separator-icon {
-        display: none;
+      &-content {
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+        text-align: center;
       }
-    }
-  }
 
-  &-group {
-    display: flex;
+      &-title {
+        font-weight: var(--token-typography-font-weight-semibold);
+        margin-top: 0.5rem;
```

Comment:
Should we be using `sizing.rems(xs)` in place of rem directly for these styles?
`_sizing.scss` should have some default sizes already defined.

Reply:
@cameronperera good catch